context_legacy.go

```diff
@@ -0,0 +1,23 @@
+// +build !go1.7
+
+package csrf
+
+import (
+	"context"
```

Comment:
It should be "github.com/gorilla/context" - goimports got confused, and
since I was running tip locally, wasn't able to test it. Thanks for
catching that though!

On Sun, May 29, 2016 at 3:06 PM Dave Cheney <notifications@github.com>
wrote:

> In context_legacy.go
> <https://github.com/gorilla/csrf/pull/35#discussion_r65012423>:
>
> > @@ -0,0 +1,23 @@
> > +// +build !go1.7
> > +
> > +package csrf
> > +
> > +import (
> > +	"context"
>
> Shouldn't this be "golang.org/x/net/context" ?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gorilla/csrf/pull/35/files/cd56b8f6fe2610c60c6a4c0329bc6e15359a94ec#r65012423>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABIcPt1L5AR-jA3uC8IlTcDRoF48ia3ks5qGg3ugaJpZM4IpOxL>
> .
>